core-devel/linux+api/spec

```diff
@@ -1,8 +1,9 @@
 VER=6.12.4
+REL=1
```

Comment:
Does fixing Anitya need such a REL bump?

Reply:
Updated to 6.12.5 since it was released yesterday.